JPetPreprocessingTask/JPetPreprocessingTask.h

```diff
@@ -0,0 +1,36 @@
+/**
+ *  @copyright Copyright 2017 The J-PET Framework Authors. All rights reserved.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may find a copy of the License in the LICENCE file.
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  @file JPetPreProcessingTask.h
+ */
+
+
+#ifndef JPETPREPROCESSINGTASK_H
+#define JPETPREPROCESSINGTASK_H
+
+#include "../JPetTask/JPetTask.h"
+#include "../JPetParams/JPetParams.h"
+#include "../JPetParamGetterAscii/JPetParamSaverAscii.h"
+
+class JPetPreprocessingTask : public JPetTask
```

Comment:
I think this task should have more meanigful name with relation to what it is actually doing.
JPEtParamBankHandler ? Think about it.
Also, add a short class description that describes what this task really does